### PR TITLE
Fixes missing api settings index

### PR DESCRIPTION
**Pull request by piyush-jena**
**Issue number:**

Closes #

**Description of changes:**
- Pulls forward docs relevant to api settings index for 1.19.0 to 1.20.0

**Terms of contribution:**

By submitting this pull request, I confirm that my contribution is made under
the terms of the licenses outlined in the LICENSE-SUMMARY file.

### CODE REVIEW

**Comment by larvacea** [comment]
I verified that the Settings Index and Settings Reference pages are populated with reasonable-looking data. I did not encounter any blank pages or obvious missing data elsewhere.

File: data/settings/1.20.x/autoscaling.toml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    "`true`",
+    "`false`"
+]
+note = """
+Only set this value to `true` if the instance is currently (or will be) in an auto scaling group. 
+"""
+default = "`false`"
+
+[[docs.ref.should-wait.example]]
+value = "true"
```
Comment (ytsssun) [inline data/settings/1.20.x/autoscaling.toml:15]: I think the team had this convention to add newline to the EOF. But I also see these are missing in past versions. Maybe starting from 1.20.x we can start enforcing this?

**Review by ytsssun — APPROVED** [review]

**Comment by ytsssun** [comment]
I also verified the change and going through the local rendering. I did not see the empty settings index now. Also don't see anything that is abnormal.

**Review by larvacea — APPROVED** [review]
Looks good to me. Thanks!

**Review by monirul — APPROVED** [review]
LGTM

File: data/settings/1.20.x/autoscaling.toml (hunk 1 above)
Comment (piyush-jena) [reply data/settings/1.20.x/autoscaling.toml]: fixed it for these toml files

**Pull request merged**